ecms_base/features/custom/ecms_emergency_notification/ecms_emergency_notification.info.yml

```diff
@@ -0,0 +1,58 @@
+name: 'Emergency Notification'
+description: 'Provides Emergency Notifications'
+type: module
+core_version_requirement: '^9.4 || ^10'
+dependencies:
+  - 'allowed_formats:allowed_formats'
+  - 'autologout:autologout'
+  - 'drupal:block_content'
+  - 'drupal:content_moderation'
+  - 'drupal:content_translation'
+  - 'drupal:contextual'
+  - 'drupal:field'
+  - 'drupal:file'
+  - 'drupal:filter'
+  - 'drupal:help'
+  - 'drupal:language'
+  - 'drupal:layout_builder'
+  - 'drupal:locale'
+  - 'drupal:media'
+  - 'drupal:menu_link_content'
+  - 'drupal:menu_ui'
+  - 'drupal:node'
+  - 'drupal:options'
+  - 'drupal:path'
+  - 'drupal:system'
+  - 'drupal:taxonomy'
+  - 'drupal:text'
+  - 'drupal:user'
+  - 'drupal:views'
+  - 'drupal:workflows'
+  - 'ecms_basic_page:ecms_basic_page'
+  - 'ecms_event:ecms_event'
+  - 'ecms_landing_page:ecms_landing_page'
+  - 'ecms_location:ecms_location'
+  - 'ecms_notification:ecms_notification'
+  - 'ecms_paragraphs:ecms_paragraphs'
+  - 'ecms_person:ecms_person'
+  - 'ecms_press_release:ecms_press_release'
+  - 'ecms_promotions:ecms_promotions'
+  - 'ecms_publications:ecms_publications'
+  - 'media_revisions_ui:media_revisions_ui'
+  - 'menu_admin_per_menu:menu_admin_per_menu'
+  - 'moderated_content_bulk_publish:moderated_content_bulk_publish'
+  - 'moderation_dashboard:moderation_dashboard'
+  - 'paragraphs:paragraphs'
+  - 'paragraphs:paragraphs_type_permissions'
+  - 'publishcontent:publishcontent'
+  - 'rabbit_hole:rabbit_hole'
+  - 'redirect:redirect'
+  - 'role_delegation:role_delegation'
+  - 'scheduled_transitions:scheduled_transitions'
+  - 'simple_menu_permissions:simple_menu_permissions'
+  - 'simple_sitemap:simple_sitemap'
+  - 'webform:webform'
+  - 'webform:webform_node'
+  - 'webform_encrypt:webform_encrypt'
```

Comment:
Why is webform listed as a dependency? There are many others that don't seem to fit here?

Reply:
The dependencies are set by the features module. I am hesitant to remove dependencies when I don't know why they are there.

Reply:
I removed excess dependencies